tox.ini

```diff
@@ -417,6 +418,8 @@ deps =
     psycopg226: psycopg2>=2.6,<2.7
     psycopg227: psycopg2>=2.7,<2.8
     psycopg228: psycopg2>=2.8,<2.9
+    pynamodb43: pynamodb>=4.3.1
+    pynamodb43: moto>=1.0,<2    
```

Comment:
Let's add older versions back to 3.0 for each minor version (4.3, 4.2, 4.1, 4.0,...). If you start running into issues for versions older than ~2 years then I would just stop at the version that breaks the test suite.